app/src/main/java/com/example/bigowlapp/model/Group.java

```diff
@@ -54,7 +55,7 @@ public void setMonitoringUserId(String monitoringUserId) {
     }
 
     public List<String> getSupervisedUserId() {
-        return supervisedUserId;
+        return (supervisedUserId == null) ? new ArrayList<>() : supervisedUserId;
```

Comment:
Great! Now an empty array will be created and inserted in "supervisedUserId" field within the proper "group" document on Cloud Firestore when an user first create an account and get a new monitoring group.